src/graph/entities/graph_entity.h

```diff
@@ -62,6 +73,12 @@ SIValue *GraphEntity_GetProperty(const GraphEntity *e, Attribute_ID attr_id);
 /* Updates existing attribute value. */
 void GraphEntity_SetProperty(const GraphEntity *e, Attribute_ID attr_id, SIValue value);
 
+/* prints the id into a buffer, returns what is the string length */
+int GraphEntity_IdToString(const GraphEntity *e, char *buffer, int bufferLen);
+
+/* prints the properties into a buffer, returns what is the string length */
+int GraphEntity_PropertiesToString(const GraphEntity *e, char *buffer, int bufferLen);
```

Comment:
Consider introducing GraphEntity_ToString, similar to node/edge_ToString

Reply:
This might not be useful since the common way to write a graph entity is
1. id
2. label or relationship type
3. properties
all of the above is written inside round or square brackets, for nodes and edges, respectively.
since a graph entity is an abstract entity I believe it is pointless for it to have its own "toString" function

Reply:
done